### DISCUSSION TITLE
引入 formily/designable-antd 还需要 手动安装 @designable/react-settings-form

### DISCUSSION THREAD

**Discussion by GadflyX** [alibaba/formily#2023] [Q&A 2.x] [answered]
想引入表单设计器，发现还需要手动安装 @designable/react-settings-form 依赖。需要手动引入 @designable/react-settings-form 依赖 ，不然会报错误 。「No matching version found for @designable/react-settings-form@^0.4.34」
     不知道这是否个 bug ？
      版本  "@formily/designable-antd": "^2.0.0-rc.1",
                "@designable/react-settings-form": "^0.4.35",
               "@formily/core": "^2.0.0-beta.22",

**Comment by janryWang** [comment, marked as answer]
就是要求手动安装
